hack/update-static.yaml.sh

```diff
@@ -27,4 +29,6 @@ cat $CRD_DIR/vulnerabilityreports.crd.yaml \
   $STATIC_DIR/02-trivy-operator.rbac.yaml \
   $STATIC_DIR/03-trivy-operator.config.yaml \
   $STATIC_DIR/04-trivy-operator.policies.yaml \
-  $STATIC_DIR/05-trivy-operator.deployment.yaml > $STATIC_DIR/trivy-operator.yaml
+  $STATIC_DIR/05-trivy-operator.deployment.yaml \
+  $STATIC_DIR/05-trivy-operator.service.yaml \
+  $STATIC_DIR/05-trivy-operator.servicemonitor.yaml \ > $STATIC_DIR/trivy-operator.yaml
```

Comment:
@Dentrax Thank you for the patience here. To wrap this up, two things only:

1 - fix the file numbers as per my review above
2 - dont' add `servicemonitor.yaml` here, or else static deploy will depend on prometheus-operator

Thank you!

Reply:
Oh, thanks! Also sorry I don't get the idea of this `sh` file, but I think It'd be better to handle these stuff in automated way such as a `make` file or smth.

Done!